lib/util.js

```diff
@@ -3,26 +3,26 @@ var utilConfig = {
 		{
 			type : "attr",
 			attrName : "ng-model",
-			match :  /[b|B]y\.model\(\s*['|"](.*?)['|"]\s*\)/gi
+			match :  /by\.model\(\s*['|"](.*?)['|"]\s*\)/gi
 		},
 		{
 			type : "attr",
 			attrName : "ng-repeat",
-			match : /[b|B]y\.repeater\(\s*['|"](.*?)['|"]\s*\)/gi
+			match : /by\.repeater\(\s*['|"](.*?)['|"]\s*\)/gi
 		},
 		{
 			type : "cssAttr", 
-			match : /[b|B]y\.css\(\s*['|"]\[(.+=.+)\]['|"]\s*\)/gi
+			match : /by\.css\(\s*['|"](.*?)['|"]\s*\)/gi
```

Comment:
This was the main change from `\[(.+=.+)\]` which required something like `name="test"` or `class="foo"`